src/sage/combinat/k_regular_sequence.py

```diff
@@ -918,6 +1026,265 @@ def _n_to_index_(self, n):
         except OverflowError:
             raise ValueError('value {} of index is negative'.format(n)) from None
 
+    def guess(self, f, n_max=100, max_dimension=10, sequence=None):
+        r"""
+        Guess a `k`-regular sequence of `(f(n))_{n\geq0}`.
```

Comment:
Perhaps
```suggestion
        Guess a `k`-regular sequence whose first terms coincide with `(f(n))_{n\geq0}`.
```
https://github.com/sagemath/sage/issues/21204#issuecomment-1417820841 Item 3